outlook/src/classes/Project.ts

```diff
@@ -0,0 +1,12 @@
+export default class Project {
+    id: number;
+    name: string;
+
+    static fromJson(o: Object): Project {
+        let project = new Project();
+        project.id = o['project_id'];
+        project.name = o['name'];
```

Comment:
Since when the user logs in, in outlook, all companies to which he has access to are added to cid's, so I think that putting the company in the response won't really be useful.

Can you tell me what you think about it?

Reply:
Ah I see now, okay then, I think that it's not harmful if we can add it for now, and see later.